src/excalidraw-app/index.tsx

```diff
@@ -446,6 +452,56 @@ const ExcalidrawWrapper = () => {
         event.type === EVENT.FOCUS
       ) {
         syncData();
+
+        const disconnect = () => {
+          disconnectTimeout = setTimeout(
+            () => collabAPI.stopCollaboration(false),
```

Comment:
I'm actually not sure this is the best UX. `stopCollaboration` is the same as if you went to the collab dialog and clicked on "stop session". It removes the link from url, and revers whiteboard to your previous localStorage content. So the tab effectively becomes a regular, non-room tab.

And on reconnect it goes through the regular flow again (loads the link, loads the content...). In prod the reconnect itself is not good UX, because loading the data from firestore takes time.

I think we should implement a custom disconnect/reconnect logic for this use case alone.